uiplib/GUI.py

```diff
@@ -211,11 +211,15 @@ def set_wallpaper(self):
         image = self.images[self.index]
         change_background(image)
 
-    def enable_subreddit(self, mainFrame):
-        sub_label = Label(mainFrame, text="Enter Subreddits")
-        sub_label.grid(row=1, column=2, padx=0, pady=0)
-        self.sub_entry = Text(mainFrame, height=10, width=20)
-        self.sub_entry.grid(row=2, column=2, padx=10, pady=10, sticky=W)
+    def toggle_subreddit(self, mainFrame):
+        if self.reddit.get() == True:
+            self.sub_label = Label(mainFrame, text="Enter Subreddits")
```

Comment:
we dont need to create object again in the `enable-disable-enable` cycle as pointed out below